### PR TITLE
Add outlining menu items and keybindings

### LINKED ISSUE #1275: [Outlining] Support for keyboard shortcuts
Support the standard VS shortcuts for outline blocks e.g. ```CTRL M + M``` collapses / expands a block.

### PR DESCRIPTION
**Pull request by dungpa**
Close #1275.

Most of menu items are working fine. Only 'Collapse to Definitions' needs special treatment.

### CODE REVIEW

File: src/FSharpVSPowerTools.Logic/Outlining/OutliningTagger.fs [hunk 1]
```diff
@@ -421,7 +423,7 @@ type OutliningTagger
                     { new IOutliningRegionTag with
                         member __.CollapsedForm      = collapseText :> obj
                         member __.IsDefaultCollapsed = collapseByDefault scope
-                        member __.IsImplementation   = false
+                        member __.IsImplementation   = true
```
Comment (vasily-kirichenko) [inline src/FSharpVSPowerTools.Logic/Outlining/OutliningTagger.fs]: why?

File: src/FSharpVSPowerTools/Commands/OutliningFilterProvider.cs [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            [Import(typeof(SVsServiceProvider))] System.IServiceProvider serviceProvider,
+            IVsEditorAdaptersFactoryService editorFactory)
+        {
+            _serviceProvider = serviceProvider;
+            _editorFactory = editorFactory;
+        }
+
+        private static void AddCommandFilter(IVsTextView viewAdapter, OutliningFilter commandFilter)
+        {
+            if (!commandFilter.IsAdded)
```
Comment (vasily-kirichenko) [inline src/FSharpVSPowerTools/Commands/OutliningFilterProvider.cs]: I guess we cannot use a singleton property here for some reason?

**Comment by vasily-kirichenko** [comment]
Cool stuff. I've tested it in VS 2015 U1 only, I don't have VS 2013 installed anywhere.

**Comment by dungpa** [comment]
Heh, I only tested VS2013 here. Will fix Collapse to Definitions a bit later.

**Comment by vasily-kirichenko** [comment]
Great, so together we can test all supported platforms :)

**Pull request merged**